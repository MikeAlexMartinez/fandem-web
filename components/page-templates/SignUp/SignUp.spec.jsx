import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import SignUpComponent from './SignUp';

describe('<SignUp />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<SignUpComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
