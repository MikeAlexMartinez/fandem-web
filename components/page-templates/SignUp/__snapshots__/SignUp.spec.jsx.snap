// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUp /> renders and matches the snapshot 1`] = `
<SignUp
  classes={Object {}}
/>
`;
